lib/_stream_writable.js

```diff
@@ -726,7 +744,20 @@ Object.defineProperty(Writable.prototype, 'writableFinished', {
   }
 });
 
-Writable.prototype.destroy = destroyImpl.destroy;
+const destroy = destroyImpl.destroy;
+Writable.prototype.destroy = function(err, cb) {
+  const state = this._writableState;
+  if (!state.destroyed) {
+    for (let entry = state.bufferedRequest; entry; entry = entry.next) {
+      entry.callback(new ERR_STREAM_DESTROYED('write'));
+    }
+    state.bufferedRequest = null;
+    state.lastBufferedRequest = null;
+    state.bufferedRequestCount = 0;
+  }
+  destroy.call(this, err, cb);
+}
```

Comment:
The linter is demanding a semi-colon after this line.

Reply:
fixed